policyengine_us/parameters/gov/states/oh/tax/income/credits/retirement_income/income_threshold.yaml

```
@@ -0,0 +1,14 @@
+description: Ohio provides the retirement income credit for filers with state AGI below this threshold.
```

Avoid abbreviations in descriptions. And threshold usually means an amount above which something is relevant, rather than below which.
```suggestion
description: Ohio limits its retirement income credit to filers with state adjusted gross income below this amount.
```